webpack/react_app/components/RecentJobsCard/RecentJobsCard.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable camelcase */
+
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Skeleton from 'react-loading-skeleton';
+import ElipsisWithTooltip from 'react-ellipsis-with-tooltip';
+
+import { Grid, GridItem } from '@patternfly/react-core';
+import {
+  OkIcon,
+  ErrorCircleOIcon,
+} from '@patternfly/react-icons/dist/js/icons';
+import {
+  PropertiesSidePanel,
+  PropertyItem,
+} from '@patternfly/react-catalog-view-extension';
+import ArrowIcon from '@patternfly/react-icons/dist/js/icons/arrow-icon';
+
+import { get } from 'foremanReact/redux/API';
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import CardItem from 'foremanReact/components/common/CardTemplate/CardItem';
```

Comment:
I really feel this import is weird, it would feel more natural to have a something like `HostDetailCardItem` or `HostDetails/cards/AccordeonCardItem`

Reply:
Thanks for updating it, now we need to update it here :)